config/fcos/v1_3/translate.go

```diff
@@ -64,6 +64,13 @@ func (c Config) ToIgn3_2Unvalidated(options common.TranslateOptions) (types.Conf
 		return types.Config{}, translate.TranslationSet{}, r
 	}
 	r.Merge(c.processBootDevice(&ret, &ts, options))
+	for _, disk := range ret.Storage.Disks {
+		for _, partition := range disk.Partitions {
+			if *partition.Label == "root" && partition.Number != 4 {
```

Comment:
I think `partition.Label` will be `nil` if no label is specified. So this needs to be e.g. 

```suggestion
			if partition.Label != nil && *partition.Label == "root" && partition.Number != 4 {
```

and similarly for the others.